data/json/npcs/factions.json

```diff
@@ -872,6 +873,7 @@
     "power": 100,
     "food_supply": 115200,
     "wealth": 75000000,
+    "price_rules": [ { "group": "games_board_all", "markup": 2.5 } ],
```

Comment:
PR text implies this should be `premium` instead of `markup`

Reply:
You're right, my bad here. First time dealing with this part of the json. I was supposed to change it to ``premium`` then forgot. I'll change it in a moment.